style-guide.md

```diff
@@ -0,0 +1,311 @@
+
+Examples:
+
+- `this-is-a-doc.md`
+- `name-of-your-image.png`
+
+Each version's pages are found in a directory named for the version. For example, pages for CockroachDB v2.0 are in the `docs > v2.1` directory. For more information about page structure, see the [Pages](https://github.com/cockroachdb/docs/blob/master/CONTRIBUTING.md#pages) section in our [Contributing Guide](https://github.com/cockroachdb/docs/blob/master/CONTRIBUTING.md). For more information about how to style page content, see [Components](#components).
+
+Each version's images are found in a directory named for the version under the `images` directory. For example, images for CockroachDB v1.0 are in the `docs > images > v1.0` directory. For more information, see [Images](#images).
+
+## Components
+
+### Code
+
+Code can be shown [inline](#inline-code) or as a [code block](#code-blocks).
+
+#### Inline Code
+
+Inline `code` has `back-ticks (``) around` it and is used when referring to code, commands, or other technical syntax within a sentence.
+
+Example: The `CREATE TABLE` statement creates a new table in a database.
+
+#### Code Blocks
+
+Code blocks are used to provide code samples, marked with an opening and closing set of 3 tildes (`~~~`). Code blocks can support syntax highlighting if you add the language name immediately after the first line of back-ticks. Shell and SQL commands should be syntax highlighted where appropriate using the following info:
+
+**Shell Code Samples**
+
+Start shell code samples with `~~~ shell` followed by a line break. The first character of the next line must be the terminal marker `$`.
+
+**SQL Code Samples**
+
+SQL code samples are broken into two sections: commands and responses.
+
+- **Commands** (e.g., `SELECT`, `CREATE TABLE`) should begin with `~~~ sql` followed by a line break. The first character of the next line must be the terminal marker `>`. Commands should be properly capitalized, and there should be only one command per code sample.
+- **Responses** (e.g., retrieved tables) should begin with `~~~` but should *not* be syntax highlighted.
+
+  Note that not all responses warrant inclusion. For example, if a SQL code sample shows `CREATE TABLE`, `INSERT`, and then `SELECT`, it's unnecessary to show the responses for `CREATE TABLE` (which is just `CREATE TABLE`) and `INSERT` (which is just `INSERT <number of rows>`).
+
+**Copy to Clipboard**
+
+Many of our code blocks are written so users can copy and paste them directly into a terminal. To make that easier, add a **Copy to Clipboard** feature by placing
+
+<pre>
+{% raw %}{<span>% include copy-clipboard.html %}{% endraw %}
+</pre>
+
+on the line directly preceding the code block, for example:
+
+<pre>
+{% raw %}{% include copy-clipboard.html %}{% endraw %}
+~~~ shell
+$ go get -u github.com/lib/pq
+~~~
+</pre>
+
+**Copy to Clipboard** should be used for every code block that can be executed.
```

Comment:
Suggestion: Put "executed" in bold. That's important info, especially for external contributors.